DOCKER.md

```diff
@@ -0,0 +1,114 @@
+
+In order to communicate with Accumulo the container will need to be able to resolve the FQDN that the servers have registered in Zookeeper. If using [fluo-uno](https://github.com/apache/fluo-uno) this is very likely the hostname of your development environment. We'll call this my.host.com and IP 192.168.0.1 for the rest of this document.
+
+### Host networking
+
+Host networking is the simplest mechanism but generally only works for linux hosts where docker has been installed on 'bare metal' e.g. through an RPM. 
+
+You can test if this will work for you by executing the following:
+```commandline
+# Start the accumulo-proxy container and enter it
+docker run -it --rm -p 42424:42424 --network="host" --name accumulo-proxy accumulo-proxy:latest bash;
+
+# Install telnet and verify if you can connect to my.host.com:9995 
+apt-get update && apt-get install telnet;
+telnet my.host.com 9995
+```
+
+If telnet can connect, then your container can resolve `my.host.com` correctly with host networking and therefore you can append `--network="host"` to your docker commands.
+
+An example of using host networking:
+```commandline
+docker run --rm -d -p 42424:42424 --network="host" --name accumulo-proxy accumulo-proxy:latest
+```
+
+Note: You do not need to map your ports (-p) if using host networking, but we include it for clarity.
+
+For more details see the official docker documentation: [Use host Networking](https://docs.docker.com/network/host)
+
+### Non-Host networking
+
+If you run outside of a single node linux installation, e.g. Docker for Mac, Docker for Windows or use a VM to isolate your docker engine then you will likely need to take this path.
+
+Docker allows you to supply additional addresses to be resolved by the container, and these are automatically added by Docker to the /etc/hosts 
+
+For each host add a `--add-host FQDN:IP` entry to your docker run command, you can add multiple entries if need to, see the official docs covering [network settings](https://docs.docker.com/engine/reference/run/#network-settings).
+
+An example of using this approach:
+
+```commandline
+docker run --rm -d -p 42424:42424 --add-host "my.host.com:192.168.0.1" --name accumulo-proxy accumulo-proxy:latest
+```
+
+## Cleanup
+Once completed you should stop and remove the container.
+```commandline
+docker stop accumulo-proxy;
+docker rm accumulo-proxy;
+```
+
+## Troubleshooting
+It can often be difficult to know where to start with troubleshooting inside containers, if you need to enter the container without starting the proxy we support this:
+```commandline
+docker run -it --rm -p 42424:42424 --network="host" --name accumulo-proxy accumulo-proxy:latest bash
+``` 
+
+The container is very slim so if need be you can add additional tools using `apt`. 
+
+If you wish to manually execute the accumulo-proxy in you can:
```

Comment:
```suggestion
If you wish to manually execute the accumulo-proxy in the container you can:
```

Reply:
Fixed in latest commit along with other aspects.